src/status_im/native_module/impl/module.cljs

```diff
@@ -54,9 +55,12 @@
 (defn init-jail []
   (when status
     (call-module
-     (fn []
-       (let [init-js (str js-res/status-js "I18n.locale = '" rn-dependencies/i18n.locale "';")]
-         (.initJail status init-js #(log/debug "jail initialized")))))))
+      (fn []
+        (let [init-js (str js-res/status-js "I18n.locale = '" rn-dependencies/i18n.locale "'; ")
+              init-js' (if config/jsc-enabled?
+                         (str init-js js-res/web3)
+                         init-js)]
+          (.initJail status init-js' #(log/debug "jail initialized")))))))
```

Comment:
Can we add if we are using JSC or not to this debug message?

Reply:
ok